### ISSUE TITLE
Audio Visualizations Downloads a Different Titled Device

### ISSUE THREAD

**Issue by EricAndrechek** [dvx/lofi#27]
The provided link to the audio device to download [here is the link](https://support.shinywhitebox.com/hc/en-us/articles/204161459-Installing-iShowU-Audio-Capture) downloads and is called eqMac2 in Audio MIDI Setup and does not work with Lofi. Also in the FAQs it says "Check out the **know** bugs or file a new issue." and I think it should say "Check out the **known** bugs or file a new issue." (change know to known)

**Comment by dvx** [comment]
Thanks for this report, I'll investigate!

**Comment by dvx** [comment]
Unfortunately, 1.5.0+ no longer supporting visualizations on Mac, to avoid dealing with this complication.

**Comment by EricAndrechek** [comment]
Hey @dvx, I just wanted to let you know I think I found a way to add music visualizations in a way that doesn't require audio access. There is this API I stumbled across called [Textyl](https://textyl.co/) which offers both realtime lyrics and beat timings for many songs. The beat timings currently aren't working for as many songs as the lyrics are, but I know the developer is working on adding them in currently. 

Pulling beat timings from this API could be a way to show visualizations on otherwise unsupported devices. Also, this could potentially allow you to add support for a lyrics display, which would be a game-changer! I don't have much experience in building native apps or anything, but in seeing that this is built on nodejs I may be able to take a look in a few months when I finish up my current project. Let me know if beat timings and lyrics interest you, and perhaps we can work together to add lyrics to this app too!
